sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -0,0 +1,210 @@
+# Migration Guide (EventHubs v2 to v5)
+
+This document is intended for users that are familiar with EventHubs V2 (`@azure/event-hubs@2.x.x` & `@azure/event-processor-host@2.x.x`) and wish 
+to migrate their application to EventHubs V5.
+
+For a more general introduction to Event Hubs, see the [readme.md](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/README.md) 
+for the @azure/event-hubs package.
```

Comment:
If this line intended to re-direct new users, I would suggest to change it to

> For users new to the JavaScript SDK for Event Hubs, please see the [readme file for the @azure/event-hubs package](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/README.md)

Reply:
I added a period for @chradek but that's a great idea.